config/routes.rb

```diff
@@ -10,7 +10,7 @@
 =end
 Rails.application.routes.draw do
 
-  mount SwaggerUiEngine::Engine, :at => '/open-api'
+  mount SwaggerUiEngine::Engine, at: '/open-api'
```

Comment:
why not rollback to `:at => '/open-api'` so that this diff does not appear at all?

Reply:
Done.